content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "How we Sign and Verify Plugins and Rules"
+linktitle: "How we Sign and Verify Plugins and Rules"
+date: 2023-10-12
+author: Massimiliano Giovagnoli, Luca Guerra
+slug: sign-verify-plugins-rules
+images:
+  - /blog/how-we-sign-and-verify-plugins-and-rules/images/falcoctl-signature-featured.png
+---
+
+## Falco v0.36.0 and the SSC security
+
+The latest stable v0.36.0, alongside falcoctl 0.6.1 and the 0.7.0 Helm chart brought new features and new improvements in the supply chain security of Falco artifacts. The two main artifacts are plugins and rule sets as you already know. They're shipped in the OCI specification format and distributed through the official Falcosecurity OCI repositories.
+
+Software supply chain attacks aim at injecting malicious code into software components like the above, to compromise downstream users. These types of attacks are among the primary threats in today's threat landscape.
+
+In particular, attackers abuse trust relationships existing between the different open-source stakeholders. The increase in attacks on open-source software throughout the last few years demonstrates that attackers consider them a viable means for spreading malware.
+
+## SSC safeguards
+
+Safeguards against supply chain attacks can be classified by control type: directive, preventive, detective, corrective, and recovery. But there ain't no such thing as a free lunch. Besides safeguards classification, the utility-to-cost ratio can also be an important factor in deciding where to start in improving the supply chain security of software, and can be pretty easy to measure it.
+
+There are cheap preventive safeguards that can be implemented in open source projects especially, where stakeholders platea can be pretty wide considering the contributions. For example, branch protection rules are usually simple per-code repository configurations in providers (e.g. GitHub) and alongside pull request-based flows enforcing code review quorum, are also standard best practices nowadays. The same applies to reproducible builds, dependency pinning, build steps isolation, MFA authentication to repository providers, etc.
+
+On the other side, safeguards with a high utility value can require a considerable amount of effort to be implemented end-to-end (producer and consumer sides).
+
+MITRE and OpenSSF increased their efforts to improve SSC security recently in general and in the open source, by providing frameworks and tools that increase the utility-to-cost ratio.
+MITRE for example proposes an end-to-end framework to preserve the integrity of the software supply chain. OpenSSF develops the SLSA framework that groups several security best practices for open-source projects.
```

Comment:
Might be nice to link to articles about the framework and a "What is SLSA" blog or something like that for readers who need more context

Reply:
@maxgio92 :point_up: can you help with the links?

Reply:
Thanks @mikegcoleman, sure @LucaGuerra

Reply:
Proposed